src/ordered-elements.js

```diff
@@ -19,9 +19,13 @@ export default class OrderedElements {
         }
     }
 
-    set(key /* : string */, value /* : any */) {
+    set(key /* : string */, value /* : any */, preserveOrder /* : ?boolean */) {
```

Comment:
Could we make `preserveOrder` default to `true` and then explicitly set it to `false` in the places where we're overriding? I think it makes more sense for something called `OrderedElements` to preserve order by default.